decidim-assemblies/spec/system/admin/valuator_checks_components_spec.rb

```diff
@@ -10,7 +10,7 @@
     decidim_admin_assemblies.components_path(assembly)
   end
   let(:components_path) { participatory_space_path }
-  let!(:user) { create :user, :confirmed, organization: }
+  let!(:user) { create :user, :confirmed, admin_terms_accepted_at: Time.current, admin: false, organization: }
```

Comment:
You can also use the `:admin_terms_accepted` trait here.
```suggestion
  let!(:user) { create :user, :confirmed, :admin_terms_accepted, admin: false, organization: }
```